test/parse.js

```diff
@@ -182,9 +196,11 @@ test('do not override a config field with value from query string', function(t){
   t.end();
 });
 
-test('return last value of repeated parameter', function(t){
-  var connectionString = 'pg:///?keepalives=1&keepalives=0';
-  var subject = parse(connectionString);
-  t.equal(subject.keepalives, '0');
-  t.end();
-});
\ No newline at end of file
+
+// test('return last value of repeated parameter', function(t){
```

Comment:
@monteslu is it really necessary this test and behavior? 

I think it's more confusing, and also I'd prefer to use URL and URL#searchParams#get (but it takes first value not last)